src/components/NcRichContenteditable/NcMentionBubble.vue

```diff
@@ -158,7 +158,10 @@ $bubble-avatar-size: $bubble-height - 2 * $bubble-padding;
 	&__select {
 		position: absolute;
 		z-index: -1;
-		left: -1000px;
+		left: -100vw;
+		width: 1px;
+		height: 1px;
+		overflow: hidden;
```

Comment:
or just `visibility: hidden` ?

Reply:
you can just use `hidden-visually` CSS class

Reply:
Or this, but you already merged)